apps/frontend/src/components/fap/Proposals/FapProposalsAndAssignmentsTable.tsx

```diff
@@ -716,6 +716,12 @@ const FapProposalsAndAssignmentsTable = ({
     })
   );
 
+  const maxPageLength = FapProposalsWitIdAndFormattedDate.length;
+
+  const pageSizeOptions = [5, 10, 20, maxPageLength]
+    .sort()
```

Comment:
Just a small thing: using the sort function on numbers without a compare function gives the wrong order. [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#description](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#description)